packages/web-runtime/tests/unit/components/SidebarNav/SidebarNavItem.spec.js

```diff
@@ -21,7 +21,7 @@ const propsData = {
   active: false,
   target: exampleNavItem.route.path,
   icon: exampleNavItem.icon,
-  index: 5,
+  index: '5',
```

Comment:
how is this related? 🤔

Reply:
It's not. It just printed a warning to the console when running unit tests, so I fixed it. ;-)